app/models/thredded/user_extender.rb

```diff
@@ -65,5 +65,11 @@ def thredded_anonymous?
     def following?(topic)
       UserTopicFollow.find_by(topic_id: topic.id, user: self)
     end
+
+    def thredded_display_name
+      send(Thredded.user_display_name_method).presence || fail(<<-ERROR)
```

Comment:
funnily enough I wrote `or`, then looked at it and thought maybe it was a perlism .